paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -67,28 +68,39 @@ internal fun PaymentMethodsUI(
                 userScrollEnabled = isEnabled,
                 modifier = Modifier.testTag(TEST_TAG_LIST)
             ) {
-                itemsIndexed(items = paymentMethods, itemContent = { index, item ->
-                    PaymentMethodUI(
-                        modifier = Modifier.testTag(
-                            TEST_TAG_LIST + stringResource(item.displayNameResource)
-                        ),
-                        viewWidth = viewWidth,
-                        iconRes = item.iconResource,
-                        title = stringResource(item.displayNameResource),
-                        isSelected = index == selectedIndex,
-                        isEnabled = isEnabled,
-                        tintOnSelected = item.tintIconOnSelection,
-                        itemIndex = index,
-                        onItemSelectedListener = {
-                            onItemSelectedListener(paymentMethods[it])
-                        }
-                    )
-                })
+                itemsIndexed(
+                    items = paymentMethods,
+                    itemContent = { index, item ->
```

Comment:
Super nit: We could use a trailing lambda here.

Reply:
Done